docs/develop/09_polarpath/01_overview.md

```diff
@@ -0,0 +1,91 @@
+---
+id: what-is-polarpath
+title: Overview
+---
+
+Polar path consists of an `asset-swap` pallet for KILT chain runtimes and extrinisics that let parachain-based chain developers to make their native token accessible on the Ethereum network using [the Snowbridge bridge](https://docs.snowbridge.network) between Polkadot and Ethereum.
+
+Polar path provides parachains with a solution for creating an ERC-20 wrapper around their native tokens, enabling a trustless conversion of any existing native parachain token (PARA) into a wrapped parachain token (wPARA) on Ethereum while maintaining the trustlessness guarantees Snowbridge provides.
+
+The pallet enables a configured conversion rate of parachain tokens on the origin chain to an ERC-20 wrapper token on Polkadot's AssetHub.
+
+## Setup
+
+:::tip Asset Hub
+The Polkadot Relay Chain doesn't natively support assets apart from DOT. Instead, parachains support this functionality, and for Polkadot, this parachain is the [Asset Hub](https://wiki.polkadot.network/docs/build-integrate-assets).
+:::
+
+Making a Parachain Token to be usable on Ethereum requires the following steps:
+
+1. Deploy an ERC-20 contract token to Ethereum with a fixed supply based on the parachain needs.
+2. [Disable the token issuer rights](https://ethereum.org/en/guides/how-to-revoke-token-access/) on the ERC-20 contract.
+3. [Register the ERC-20 token on Asset Hub](https://wiki.polkadot.network/docs/learn-assets#creation-and-management).
```

Comment:
Wrong link. Registration on AH happens via the bridge: https://docs.snowbridge.network/applications/token-transfers#token-registration.

Reply:
Looks good!